src/profile-logic/marker-styles.js

```diff
@@ -171,10 +170,18 @@ const markerStyles: { +[styleName: string]: MarkerStyle } = {
   },
   GCSlice: gcStyle,
   GCMinor: gcStyle,
-  CCSlice: gcStyle,
+  'GC Interrupt': gcStyle,
```

Comment:
This is a new marker added in https://bugzilla.mozilla.org/show_bug.cgi?id=1752646